mycroft/skills/npr_news/test/intent/sample2.intent.json

```diff
@@ -1,7 +1,7 @@
 {
-  "utterance": "play the news",
+  "utterance": "is there any news",
```

Comment:
Yeah, there was a conflict between `play` and `news` as keywords in the `play the news` utterance. That's a larger problem with intent parsing in general, and can't really be solved here. I suggested that he just change that test for the moment, since the play part of it isn't an essential part of the test.